src/AccountProfileOverlay.jsx

```diff
@@ -71,6 +80,38 @@ const overlay = (
+              </svg>
+              <VerificationText secondary>Human by <a href={'https://' + verifications.human_provider + '/'} target="_new">{verifications.human_provider}</a></VerificationText>
+            </div>
+        )}
+        {verifications.kyc_provider && (
+            <div>
+              <svg width="20" height="20" viewBox="0 0 30 30" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
You can try this icon:
```suggestion
              <i class="ph-duotone ph-seal-check"></i>
```
and play with color or skip this comment

Reply:
![Screenshot 2023-09-18 at 5 23 10 PM](https://github.com/near/near-discovery-components/assets/671506/d8cc7cf4-18c8-4e6a-9a0e-2f49ad2b4565)

That's a cool idea. It looks like the icon is square though so it might be a large amount of work to fill in the colors. 

Something for us to think about @xuan-ytr https://phosphoricons.com/